docs/ios-client/onboarding.md

```diff
@@ -20,32 +18,25 @@ Download Xcode from the [App Store](https://apps.apple.com/us/app/xcode/id497799
 
 ## Install Xcode Command Line Tools
 
-Open Xcode and you should be prompted with installing the command line tools, or  run this in a Terminal window:
+Open Xcode and you should be prompted with installing the command line tools, or run this in a Terminal window:
 
 ```sh
 xcode-select --install
 ```
 
-## Install CocoaPods and CocoaPods-Patch
+## Install Ruby
```

Comment:
Installing Ruby is only needed for the subsequent optional Fastlane section, right? If so, we should just put the (important) note about non-system Ruby in the subsequent section, since readers not intending to use Fastlane can avoid dealing with Ruby altogether.